### PR TITLE
Docs: Revamp Extraction Use Case

### PR DESCRIPTION
**Pull request by eyurtsev**
Still a draft wrapping up parsing and then doing a typo pass through

### CODE REVIEW

**Comment by eyurtsev** [comment]
@ccurme  no need to review just a FYI

File: docs/docs/use_cases/extraction/how_to/examples.ipynb [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7f59a745-5c81-4011-a4c5-a33ec1eca7ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "examples = [\n",
+    "    (\n",
+    "        \"The ocean is vast and blue. It's more than 20,000 feet deep. There are many fish in it.\",\n",
+    "        Person(name=None, height_in_meters=None, hair_color=None),\n",
```
Comment (ccurme) [inline docs/docs/use_cases/extraction/how_to/examples.ipynb:251]: In this case the negative sample seems to hinge on all the attributes of `Person` being optional. Is it worth including another case-- maybe with `Data`, which appears unused in this notebook?

**Review by ccurme — COMMENTED** [review]
Looks great!

File: docs/docs/use_cases/extraction/how_to/_category_.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+label: 'How-To Guides'
```
Comment (baskaryan) [inline docs/docs/use_cases/extraction/how_to/_category_.yml:1]: nit: call these "How to" elsewhere

File: docs/docs/use_cases/extraction/guidelines.ipynb
Comment (baskaryan) [inline docs/docs/use_cases/extraction/guidelines.ipynb:1]: Guidelines bit awkward, maybe "Best practices"?

File: docs/docs/use_cases/extraction/index.ipynb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    "\n",
+    "This guide will show you how to use LLMs for extraction applications!\n",
+    "\n",
+    "## Approaches\n",
+    "\n",
+    "There are 3 broad approaches for information extraction using LLMs:\n",
+    "\n",
+    "- **Tool/Function Calling** Mode: Some LLMs support a *tool or function calling* mode. These LLMs can structure output according to a given **schema**. Generally, this approach is the easiest to work with and is expected to yield good results.\n",
+    "\n",
+    "- **JSON Mode**: Some LLMs are can be forced to output valid JSON. This is similar to **tool/function Calling** approach, except that the schema is provided as part of the prompt. Generally, this approach expected to perform worse than a **tool/function calling** approach.\n",
```
[the remark this answers is not in the document]
Comment (ccurme) [reply docs/docs/use_cases/extraction/index.ipynb:37]: We might clarify the rationale, e.g., "this approach is expected to perform worse than a **tool/function calling** approach, as adherence to the schema is not enforced."

File: docs/docs/use_cases/extraction/index.ipynb
Comment (baskaryan) [inline docs/docs/use_cases/extraction/index.ipynb:1]: should put this higher up in side bar

File: docs/docs/use_cases/extraction/index.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (eyurtsev) [reply docs/docs/use_cases/extraction/index.ipynb:37]: Ah source is intuition should probably explain why or change language

File: docs/docs/use_cases/extraction/how_to/examples.ipynb (hunk 1 above)
Comment (hwchase17) [reply docs/docs/use_cases/extraction/how_to/examples.ipynb:251]: i think data should be used down below

**Comment by eyurtsev** [comment]
This looks good. Merging and making another PR to make a few minor corrections.

**Pull request merged**